concent_api/gatekeeper/views.py

```diff
@@ -40,9 +46,11 @@ def download(request):
     init_messages()
     response = parse_headers(request)
     if response is not None:
+        logger.warning(response.content.decode())
         return response
+    logger.info('Request passed all download validations.')
 
-    return HttpResponse("", status = 200)
+    return HttpResponse("Request passed all download validations.", status = 200)
```

Comment:
All responses should be in the JSON format.

Reply:
Okey, now it's `return JsonResponse({"message": "Request passed all download validations."}, status = 200)`